code/_onclick/hud/hud.dm

```diff
@@ -187,3 +187,5 @@
 	else
 		to_chat(usr, "<span class ='warning'>This mob type does not use a HUD.</span>")
 
+/image
+	plane = FLOAT_PLANE // I have no fucking clue why this isn't FLOAT_PLANE by default in BYOND.
```

Comment:
Some hud objects have `/image` overlays attached to them.

To complicate things even further, items often have overlays of their own. Without this, all the overlays would have to be rebuilt with the plane set to the appropriate value when moving between hud and world.